docs/.vuepress/theme/styles/code-group.styl

```
@@ -24,3 +24,14 @@
 div[class*="language-"]::before {
   content: '';
 }
+
+div[class*="language-"] .highlight-lines {
```

Is there a way to add custom class to example block? 

```suggestion
div[class*="language-"] .highlight-lines.highlight-diff {
```

Usage
```
```example { highlight={2,4} class=.highlight-diff }

```